chain/transaction.go

```diff
@@ -31,68 +31,71 @@ var (
 type Transaction struct {
 	Base *Base `json:"base"`
 
-	Actions []Action `json:"actions"`
-	Auth    Auth     `json:"auth"`
+	Actions Actions `json:"actions"`
+	Auth    Auth    `json:"auth"`
 
-	digest    []byte
+	preimage  []byte
```

Comment:
could we rename this field to `unsignedBytes` and the accessor function to `UnsignedBytes()` ?

Reply:
done